test/config/ca.json

```diff
@@ -35,6 +35,7 @@
       "NumSessions": 2
     }],
     "expiry": "2160h",
+    "backdate": "1h",
```

Comment:
IIUC, I'm not actually supposed to change this file yet? Maybe it is only supposed to get changed during the merge? Not sure.